rosidl_typesupport_c/resource/srv__type_support.cpp.em

```diff
@@ -94,7 +97,7 @@ static const rosidl_service_type_support_t @(spec.srv_name)_service_type_support
 
 }  // namespace rosidl_typesupport_c
 
-}  // namespace srv
+}  // namespace @(subfolder)
 
 }  // namespace @(spec.pkg_name)
 
```

Comment:
I think L105 should use `@(subfolder)`.

https://github.com/ros2/rosidl_typesupport/blob/24d917c5fea5a1a46400c9d026c032bf42a49cd0/rosidl_typesupport_c/resource/srv__type_support.cpp.em#L105

should be

```
#include "@(spec.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.srv_name))__@(list(type_supports)[0]).h"
```